engine/api/src/main/java/io/deephaven/engine/table/Table.java

```diff
@@ -308,6 +308,11 @@ public interface Table extends
     @ConcurrentMethod
     Table formatColumnWhere(String columnName, String condition, String formula);
 
+    @ConcurrentMethod
+    Table formatDataBar(String column, String valueColumn, String axis, Double min, Double max,
```

Comment:
I could actually just move it since the enums in ColumnFormatting were meant for the type hints but as shown in the comment above the param specific hints aren't working properly anyways